aggregator/src/core.rs

```diff
@@ -241,17 +243,18 @@ pub(crate) fn extract_hash_cells(
                     let row = keccak_config.set_row(&mut region, offset, keccak_row)?;
 
                     if cur_preimage_index.is_some() && *cur_preimage_index.unwrap() == offset {
-                        // 10-th column is Keccak input in Keccak circuit
-                        hash_input_cells.push(row[10].clone());
+                        hash_input_cells.push(
+                            row[keccak_packed_multi::get_input_bytes_col_cell_manager() + 4]
```

Comment:
define 5 as a const, e.g., `NUM_KECCAK_TABLE_COLUMN` and change this line to
```
keccak_packed_multi::get_input_bytes_col_cell_manager() + NUM_KECCAK_TABLE_COLUMN -1
```
(or simply `row.last().unwrap()`?)